src/convert.jl

```diff
@@ -0,0 +1,229 @@
+module Conversion
+
+    using ..SAM
+    using ..BAM
+
+    using BioSequences, BioAlignments
+    import Base.convert
+
+    global  const op_to_code = Dict(
+        'M' => 0x00,
+        'I' => 0x01,
+        'D' => 0x02,
+        'N' => 0x03,
+        'S' => 0x04,
+        'H' => 0x05,
+        'P' => 0x06,
+        '=' => 0x07,
+        'X' => 0x08,
+        'B' => 0x09,
+        '0' => 0x0a
+    )
+
+    function parse_cigar(record::SAM.Record)
```

Comment:
I haven't found anything, there's just a `cigar` method that returns the cigar string. It would be good to add a `cigar_rle` method for SAM, but I went for the minimum thing I needed here.